lib/taskana-core/src/test/java/acceptance/ArchitectureTest.java

```diff
@@ -153,6 +153,20 @@ void exceptionsShouldBePlacedInExceptionPackage() {
     myRule.check(importedClasses);
   }
 
+  @Test
+  void exceptionsShouldNotHaveToStringMethod() {
```

Comment:
I also expect a rule which checks, that TaskanaException and TaskanaRuntimeException contain a `toString` method